spectral_cube/spectral_cube.py

```diff
@@ -3636,3 +3698,12 @@ def mask_channels(self, goodchannels):
 
         return self._new_cube_with(mask=mask,
                                    goodbeams_mask=goodchannels & self._goodbeams_mask)
+
+
+def _compound_region(region_list):
```

Comment:
compound regions belong in the `regions` package rather than here; is your intent to move it there eventually?

Reply:
ya..it is just for now.

Reply:
If we want this method in regions, we better get this merged to regions/master before we merge this?

Reply:
I think has this method has a very specific purpose (to convert a region list to a single `Region` object),
so do you think there is a good need to move to regions (having second thoughts here.)

Reply:
You're right, the function is more about manipulating a list into a region.  Change the function name to `_regionlist_to_single_region` and leave it here.